tools/rum-models-generator/Sources/RUMModelsGeneratorCore/Input/Swift/SwiftType.swift

```diff
@@ -54,6 +54,12 @@ internal struct SwiftEnum: SwiftType {
 
 internal struct SwiftStruct: SwiftType {
     struct Property: SwiftType {
+        enum Mutability: Int {
+           case immutable
+           case mutableInternally
+           case mutable
+        }
```

Comment:
As the order here matters a lot (changing it would be destructive), how about adding simple unit test for ensuring it or at least adding a comment to watch it?

Reply:
Totally!